app/controllers/issues/evidence_controller.rb

```diff
@@ -1,4 +1,15 @@
-class Issues::EvidenceController < AuthenticatedController
-  include MultipleDestroy
-  include ProjectScoped
+class Issues::EvidenceController < IssuesController
+  def new
+    @auto_save_key =  if params[:template]
+                        "project-#{current_project.id}-issue-#{params[:issue_id]}-evidence-#{params[:template]}"
+                      else
+                        "project-#{current_project.id}-issue-#{params[:issue_id]}-evidence"
+                      end
```

Comment:
This would be better suited for a before filter. You don't even need to assign the filter itself. If the method name is `set_auto_save_key` it will automatically run because IssuesController calls the before filter.